controllers/productController.js

```diff
@@ -0,0 +1,13 @@
+const productService = require('../services/productService');
+const { catchAsync } = require('../utils/error');
+
+const getProductByMainpage = catchAsync(async (req, res) => {
+
+  const products = await productService.getProductByMainpage();
+
+  res.status(200).json({ data : products });
+});
```

Comment:
변수 네이밍 한번만 수정해주세요!

service, dao 단에서도 수정해주시면 되겠습니다~!
```suggestion
const getProducts = catchAsync(async (req, res) => {
  const products = await productService.getProducts();

  res.status(200).json({ data : products });
});
```